app/views/admin/campaigns/index.html.haml

```diff
@@ -18,16 +18,16 @@
       %tbody
         - @campaigns.each do |campaign|
           %tr
-            %td{'id'=> "name_#{campaign.id}"}
+            %td{ 'id' => "name_#{campaign.id}" }
```

Comment:
Oh yes we can. Making the changes. But we need to work out in a differnet way @ChrisBr says it would be better if the PR has 200 or less LOC. I too think it is a good idea